### PR TITLE
Fix missing percentage sign in current position in case user balance is missing

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/currentPosition/CurrentPosition.tsx [hunk 1]
```diff
@@ -121,11 +121,11 @@ const CurrentPosition: FC<CurrentPositionInProps> = ({ language }) => {
 
   const unstakedPrincipalsPercentage = useMemo(() => {
     if (!userPrincipalsBalance || !lpTokenPrincipalReturnBalance) {
-      return 0;
+      return '0%';
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/currentPosition/CurrentPosition.tsx:124]: we should use `NumberUtils.formatPercentage`

**Review by danieledamiani — COMMENTED** [review]
a comment

File: packages/tempus-client_v2/src/components/currentPosition/CurrentPosition.tsx (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/currentPosition/CurrentPosition.tsx:124]: Makes sense, I will do that now!

Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/currentPosition/CurrentPosition.tsx]: Fixed

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**